main.js

```diff
@@ -18,6 +18,10 @@ gameState.players = [
   createPlayer({ name: 'player1' }),
   createPlayer({ name: 'player2' }),
 ];
+// set id of players
+gameState.players.forEach((player, index) => {
```

Comment:
Why do we do this here and in `highestRolling` ?
Shouldn't we put this into a rule of its own at that point and have it happen _after_ `highestRolling` ?

Reply:
Oh, we could do that. I did it in two places because we may not want to run the highestRolling rule. A separate rule is a much better idea!